src/box/box.cc

```diff
@@ -849,6 +849,26 @@ box_check_sql_cache_size(int size)
 	return 0;
 }
 
+static inline bool
+is_exp_of_two(unsigned n)
```

Comment:
Personally I'd extract this helper to trivial/util.h
As a rule we don't place extra space after bracket (even though it improves readability).